src/swap/components/SwapProvider.tsx

```diff
@@ -120,10 +133,11 @@ export function SwapProvider({
         statusName: 'init',
         statusData: {
           isMissingRequiredField: false,
+          maxSlippage: experimental.maxSlippage ?? 3,
         },
       });
     }
-  }, [hasHandledSuccess, lifeCycleStatus.statusName]);
+  }, [hasHandledSuccess, lifeCycleStatus.statusName, experimental.maxSlippage]);
```

Comment:
This introduces a bug. Think on this.
If you don't know what it is, I will tell you what bug it is tomorrow.

Reply:
Hmm probably the infinite loop bug? Will look more into this tomorrow